src/dist/replication/meta_server/greedy_load_balancer.cpp

```diff
@@ -119,6 +120,18 @@ void greedy_load_balancer::register_ctrl_commands()
         "lb.get_balance_operation_count [total | move_pri | copy_pri | copy_sec | detail]",
         "get balance operation count",
         [this](const std::vector<std::string> &args) { return get_balance_operation_count(args); });
+
+    _ctrl_balancer_ignored_apps = dsn::command_manager::instance().register_app_command(
+        {"lb.ignored_app_list"},
+        "lb.ignored_app_list [app_id1,app_id2.. | CLEAR](if no argument means get current ignored "
+        "list)",
+        "get or update balance ignored_app_list",
+        [this](const std::vector<std::string> &args) {
+            if (args.empty()) {
+                return get_balancer_ignored_app_ids();
+            }
+            return update_balancer_ignored_app_ids(args);
+        });
```

Comment:
The command semantics seem a bit of strange, how about:
lb.ignored_app_list get  # to get ignored app ids
lb.ignored_app_list set (x,x,x) # to set ignored app ids
lb.ignored_app_list clear  # to clear ignored app ids

Reply:
This is  coincident with other `remote command` : `empty` means `get current value`, `valid argument` means `set currnet value`, `clear or default`  means recovery, I think this rule may no need change.

Reply:
> lb.ignored_app_list get # to get ignored app ids
lb.ignored_app_list set x,x,x # to set ignored app ids
lb.ignored_app_list clear # to clear ignored app ids

It's a simple semantics rule to obey, `<topic> <action> [parameters]`.
I think it's not needed to keep with other commands, and we could update old commands if we found a better solution.